src/modules/root-container/root-container.view.ts

```diff
@@ -125,6 +126,10 @@ class RootContainerView extends View<IRootContainerViewStyles>
     }
   }
 
+  setRtl(rtl: boolean) {
+    this._$rootElement.setAttribute('playable-dir', rtl ? 'rtl' : 'ltr');
```

Comment:
Can we name this attribute `data-playable-dir`?

Reply:
easy